packages/grafana-schema/src/index.gen.ts

```diff
@@ -111,16 +111,5 @@ export type {
 // TODO generate code such that tsc enforces type compatibility between raw and veneer decls
 export type { LibraryPanel } from './veneer/librarypanel.types';
 
-// Raw generated types from Preferences kind.
-export type {
-  Preferences,
-  QueryHistoryPreference,
-  CookiePreferences,
-  NavbarPreference
-} from './raw/preferences/x/preferences_types.gen';
-
-// Raw generated enums and default consts from preferences kind.
-export { defaultNavbarPreference } from './raw/preferences/x/preferences_types.gen';
-
```

Comment:
^^^ levitate breaking change

BUT, this is not used anywhere -- we are now using the RTK client from swagger.